src/app/components/home-page/LatestNews.tsx

```diff
@@ -215,35 +217,14 @@ const LatestNews = () => {
               </p>
             </article>
           </li>
-          <li>
```

Comment:
Rather than adding/removing html to update what do you think about making a json object to hold the data and this populates a component?

Reply:
We'll eventually (some time before next release hopefully) do https://www.ebi.ac.uk/panda/jira/browse/TRM-28342 so I guess this will be tackled then and will be coming from an object coming from the network